addons/skin.confluence/720p/VideoOSD.xml

```diff
@@ -172,6 +172,7 @@
 			<control type="image" id="2200">
 				<width>270</width>
 				<texture>-</texture>
+				<visible>VideoPlayer.HasMenu</visible>
 				<visible>!VideoPlayer.Content(LiveTV)</visible>
```

Comment:
I dont know how to join those, when I did it didnt work hence the negated and normal stayed in separate lines.

<visible>VideoPlayer.HasMenu</visible>
<visible>!VideoPlayer.Content(LiveTV)</visible>